### ISSUE TITLE
Sample Math on filters is wrong

### ISSUE THREAD

**Issue by Willv19** [AlexsLemonade/refinebio#509]
### Context

https://github.com/AlexsLemonade/refinebio-frontend/issues/227

### Problem or idea

For some reason the filters seem to double all the numbers.

### Solution or next step

It looks like the `SearchAndFilter` view's code is wrong. That might be the first place to look. This might also require a conversation about what number to display once the PR lands about multiple filters being applied in the same category.

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.

**Comment by Miserlou** [comment]
I can't reproduce this, maybe it was confusion about samples/experiment count?

![screen shot 2018-08-23 at 11 54 09 am](https://user-images.githubusercontent.com/139987/44536815-4c17cc00-a6cb-11e8-986f-88b325ec9a43.png)

?
